docs/UserGuide.md

```diff
@@ -48,9 +48,10 @@ This user guide provides in-depth descriptions of all the available features in
 1. Ensure you have Java `11` or above installed in your Computer.
 
 1. Download the latest `guestbook.jar` from [here](https://github.com/AY2223S1-CS2103T-W16-1/tp/releases).
-
+   ![Ui](images/download.png)
```

Comment:
Should we make the image neater by using proper arrows? For reference, I think the UiComponentLabels.png is a good reference by drawing a box and marking "Click this to download". In this case, we could perhaps be more concise in doing so and we can remove download2.png as well? Also, should we capitalise the file name to be more aligned with our other files?